src/widgets/settingspages/KeyboardSettingsPage.cpp

```diff
@@ -40,13 +40,6 @@ KeyboardSettingsPage::KeyboardSettingsPage()
             auto newHotkey = dialog.data();
             int vectorIndex = getApp()->hotkeys->hotkeys_.append(newHotkey);
             getApp()->hotkeys->save();
-
-            // Select and scroll to newly added hotkey
-            auto modelRow = model->getModelIndexFromVectorIndex(vectorIndex);
-            auto modelIndex = model->index(modelRow, 0);
-            view->selectRow(modelRow);
-            view->getTableView()->scrollTo(modelIndex,
-                                           QAbstractItemView::PositionAtCenter);
```

Comment:
previously this add action would scroll further than needed to put new item at the center. Decided to not preserve this and keep all tables scroll-to-new behave the same. 
Both approaches are as good when it comes to visibility of newly added item.